app/components/spin-activity.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  scores: [],
+  actions: {
+    setScore(score) {
+      this.get('scores').push(score);
+      this.nextQuestion();
+    }
+  },
+  init() {
+    this.set('question', this.get('questions.firstObject'));
```

Comment:
Ember uses `this.get` and `this.set` to control values that are bound between its classes. The changes to this `question` will propagate to the template, akin to `$scope.question`.